tests/integration/sdk_api/artifacts/test_artifacts.py

```diff
@@ -55,3 +55,37 @@ def test_list_artifacts_filter_by_kind(self):
             project=prj, category=mlrun.api.schemas.ArtifactCategories.dataset
         )
         assert len(artifacts) == 1, "bad number of dataset artifacts"
+
+    def test_store_artifact_with_empty_dict(self):
```

Comment:
We should always strive for having a functionality being tested by the fastest possible test:
* **System** tests are the slowest one - in order to run a test correctly you need to build a whole version out of the code, transplant it in the system etc..
* **Integration** tests are faster - in order to run a test you (done automatically when you run it) build a docker image of the API, run it, and do actions in the client
* **Unit** tests are the fastest - all you need to run them is to install all requirements

Your change here is in the API code, client is irrelevant, therefore the correct place IMO to test it is the API tests, which are unit tests that bring up a mock API inside the test process

Reply:
So you suggesting removing those tests?